README.rst

```diff
@@ -10,6 +10,10 @@
 |                      | NI-ModInst | |ModInstLatestVersion| |ModInstPythonVersion| |
 |                      +------------+-----------------------------------------------+
 |                      | NI-SWITCH  | |SwitchLatestVersion| |SwitchPythonVersion|   |
+|                      +------------+-----------------------------------------------+
+|                      | NI-FGEN    | |FgenLatestVersion| |FgenPythonVersion|       |
```

Comment:
another cleanup task: besides fixing order, perhaps we should clear all these variables (FgenPythonVersion, ModInstLatestVersion) and use the module names exactly as prefixes (nimodinstLatestVersion). Anything else seems arbitrary.

Reply:
Changed